### ISSUE TITLE
The bottom right point of each rating star looks glitchy

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-frontend#7191]
### Describe the problem and steps to reproduce it:

- Log in
- Go to any add-on detail page

### What happened?

The bottom right point of the rating star looks glitchy

<img width="436" alt="screenshot 2018-12-10 14 38 10" src="https://user-images.githubusercontent.com/55398/49760227-83fe4b80-fc89-11e8-802e-05266ccd20e8.png">

### What did you expect to happen?

The bottom right point should be rounded like the other points

### Anything else we should know?

Tested on Nightly, Release and in prod and dev.

**FOR QA** Please take a look for regressions in all icons (and other SVG images). The fix to this involved changing the way we optimize SVG files.

**Comment by willdurand** [comment]
>  I'll try to find a workaround.

Fixing https://github.com/mozilla/addons-frontend/issues/5925 would likely fix this issue. Other SVGs _might_ be impacted by SVGO, but if we can't find any other visual issue, I'd say the patch for #5925 would look like a good start.

**Comment by kumar303** [comment]
Switching to inline SVGs is interesting because those won't get any optimizations at all. Just something to consider.

**Comment by ioanarusiczki** [comment]
@kumar303 I verified on all pages (I hope I covered them all). So far, I do not see visual regressions on AMO homepage, Extension, Themes tabs and categories, collections, search results, the header and footer of the page, detail pages, all reviews page, my profile page. 
I also checked the size of the icons with Inspector - the logo+title has is different in size but visually it looks the same. 

AMO dev

![header title on prod](https://user-images.githubusercontent.com/33448286/51252689-26a29c00-19a5-11e9-83e4-1322821c42bc.png)

AMO prod

![header content on prod](https://user-images.githubusercontent.com/33448286/51252718-34f0b800-19a5-11e9-9186-dd8f122fd976.png)

Also the rating stars don't display that UX bottom right issue

![ux rating stars](https://user-images.githubusercontent.com/33448286/51252868-9749b880-19a5-11e9-9792-a0fe343debb6.png)

Verified on AMO dev - FF64(Win10).
